tests/test_adjointJ.py

```diff
@@ -17,7 +17,7 @@ def test_acousticJ(dimensions, space_order):
     spacing = [15. for _ in dimensions]
```

Comment:
you see this should be called ``shape`` not ``dimensions``

Reply:
See above.